action.yml

```diff
@@ -28,6 +28,9 @@ inputs:
   codedeploy-deployment-group:
     description: "The name of the AWS CodeDeploy deployment group, if the ECS service uses the CODE_DEPLOY deployment controller. Will default to 'DgpECS-{cluster}-{service}'."
     required: false
+  codedeploy-deployment-description:
+    description: "A description of the deployment, if the ECS service uses the CODE_DEPLOY deployment controller. Will default to 'DgpECS-{cluster}-{service}'."
```

Comment:
I have a question that why do we need to set default description as `DgpECS-{cluster}-{service}`. Is there any particular need to do so?

Reply:
I would go further and say we should _not_ set any value by default as this will change behavior for existing users who would not be using this new input.

Reply:
It's been a while since I did it so can't remember the exact thinking behind it! But yes, I definitely agree. Keeping it blank if your set makes far more sense. I'll make that change.

Reply:
Looked again at the code and I'm doing what @allisaurus suggests. The description isn't defaulted to anything, I think that's a screw up by me in the parameter description. Sorry. If the description is not passed to the action, then it will not be passed to the API call.